### ISSUE TITLE
Add support for Savage Firmware (FaceID)

### ISSUE THREAD

**Issue by iambenmitchell** [encounter/futurerestore#58]
Hello, this tool hasn't been updated to support the Savage Firmware added by apple.

It is used for FaceID. Meaning that if you future restore, although your phone will work perfectly well, FaceID will be broken.

The fix should be as simple as duplicating the MAV/ICE code and modifying it for Savage.

**Comment by ghost** [comment]
Can you do this

**Comment by iambenmitchell** [comment]
No, this is way out of my league sorry. And I don’t have all the stuff to compile it
